client/src/components/Icons.tsx

```diff
@@ -32,14 +32,7 @@ export const CheckIcon = (props: SVGProps<SVGSVGElement>) => (
 );
 
 export const ChevronIcon = (props: SVGProps<SVGSVGElement>) => (
-  <svg
-    width="11"
-    height="7"
-    viewBox="0 0 11 7"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-    {...props}
-  >
+  <svg fill="none" viewBox="0 -2 10 10" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
did you change the viewBox value manually? if so, how did you know what values to set it to